### PR TITLE
mrc-2316 Fetch countries from cometr

### PR DESCRIPTION
**Pull request by EmmaLRussell**
This branch adds a backend endpoint to fetch the list of countries from cometr, and frontend fetch to get countries on load index page. 

Future tickets tickets will allow user to select country from a drop-down, and manage default population values based on country (tbd where those come from I think!)

### CODE REVIEW

File: src/app/src/main/kotlin/org/imperial/mrc/comet/controllers/DataController.kt [hunk 1]
```diff
@@ -13,6 +14,12 @@ class DataController(
     private val logger: Logger = LoggerFactory.getLogger(DataController::class.java),
     private val apiClient: APIClient
 ) {
+    @GetMapping("/countries")
+    fun countries(): ResponseEntity<String> {
+        logger.info("countries")
```
Comment (mwoodbri) [inline src/app/src/main/kotlin/org/imperial/mrc/comet/controllers/DataController.kt:19]: I vaguely remember playing with a filter to automatically log requests + params. Will dig it out in a maintenance ticket.

File: src/app/src/main/kotlin/org/imperial/mrc/comet/controllers/DataController.kt [hunk 2]
```diff
@@ -13,6 +14,12 @@ class DataController(
...
+        return apiClient.countries()
```
Comment (mwoodbri) [inline src/app/src/main/kotlin/org/imperial/mrc/comet/controllers/DataController.kt:20]: I noticed that this endpoint returned a "TEST" country. We'll presumably remove this before final release.

**Review by mwoodbri — APPROVED** [review]
Nothing at all to quibble with here... uncontroversial change but a very clean PR nevertheless!

File: src/app/src/main/kotlin/org/imperial/mrc/comet/controllers/DataController.kt (hunk 2 above)
Comment (EmmaLRussell) [reply src/app/src/main/kotlin/org/imperial/mrc/comet/controllers/DataController.kt:20]: Ah good spot, yes - or we'll need to filter out countries whose `public` flag is false, either in front or back end. To be clarified!

**Pull request merged**